dev-aws/kafka-shared/iam_dpd.tf

```diff
@@ -0,0 +1,50 @@
+resource "kafka_topic" "iam_dpd_v1" {
+  name               = "auth-customer.iam-dpd-v1"
+  replication_factor = 3
+  partitions         = 10
```

Comment:
currently there is no benefit for more than 1 partitions, so if reducing this helps the cluster by any means, we can do it